README.md

```diff
@@ -368,7 +368,7 @@ def test_reboot_instance(aws_client):
 ```
 
 By using the [built-in Python module to mock objects][pymock], we can mock the
-EC2 client and assert we edo indeed call the appropriate method with the right
+EC2 client and assert that we do indeed call the appropriate method with the right
```

Comment:
Appreciated :) Though it's usually better to have distinct PRs for these things :)

Reply:
noted for next time 😄